package.json

```diff
@@ -98,8 +98,8 @@
     "core-js-compat": "npm:3.31.0",
     "electron-to-chromium": "npm:1.4.441",
     "glob-watcher/chokidar": "npm:^3.4.0",
-    "@types/babel__core": "link:./nope",
-    "@types/babel__traverse": "link:./nope",
+    "@types/babel__core": "link:./.husky",
+    "@types/babel__traverse": "link:./.husky",
```

Comment:
`./nope` will generate a symbolic link pointing to an invalid location and gulp v5 will throw.
Here we use an arbitrary non-package directory instead.